### PR TITLE
GAWB-3213 auto-update hashes in README and CHANGELOGs

### PR DESCRIPTION
**Pull request by jmthibault79**
**PR checklist**
- [x] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [x] Bump the version in `project/Settings.scala` `createVersion()`
- [x] Update `CHANGELOG.md` for those libraries
- [x] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

After merging, _even if you haven't bumped the version_: 
- [ ] Update `README.md` and the `CHANGELOG.md` for any libs you changed with the new dependency string. The git hash is the same short version you see in GitHub (i.e. seven characters). You can commit these changes straight to develop.

### CODE REVIEW

**Review by rtitle — APPROVED** [review]
Looks great

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,10 @@ env:
...
+  - secure: Poow3t4dWza2XvLGbBWjy43mgEPDXuhWAwOQyT0ggYX8hsYaPx7byO/xIYdNKDVU3NBkfnueyGt0EavHhUEXFUwpADr/Qg6CqqyQQSY0AZCXsLwEWYuBHfBZxmgAnxw5venWtcY8jXWwavZqRfC/q5tViSq4IIuh/jvZJ1OXJ6OkApZ3p6SYTXJdJ7Qf4sKfoeqKMZ+D0RVnVF+2tywSjUicQpDk1oUZulpe4IPxquZA22NAAFCepXEiQytQ7UE7ze8vqfrADYnsK2TJeUa3EXMBCE2YQ7mXPca2TgN9UtlTikdyRmY2SLi1e2MJfW/0ZMqr+rNyzN5kWMWTGJKqbOCGcTta1Ro61aZh/xmMUv+qq2hA3oGw6EZsFe51xh/pEkzo2TVikxQyWdJHoCQZJ7Cxf+eoIlTbm17YriL1G8cpmy4lAboZSk5DTd/s3Jg0lH21auCE0O5NPQOjg/N1OPQ8GN1kXcy0MCeAnRBaTyEwqlsYaNr5ZIi3uCvblQPcqeW5+4DoEcC9mRoG1cU3ms/4do7eVXSPlzAr1+QsXCTHc6sSEX9b3UdGrDDGk89YvvFyLphOHwdC054bCvGskfc/Eh0A5R4U2oLGqcuBF2Gk9PVpOjiEduYu0KIQT4uoJeRvCpNT/6pbJMc7yGmVdSDmqs5PFBF4gkdtK+Ed6dk=
```
Comment (jmthibault79) [inline .travis.yml:12]: This is `GH_TOKEN`.  It's a *personal* github token so we should replace this with something more appropriate soon.

File: scripts/version_update.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  for f in README.md */CHANGELOG.md
+    do
+      echo "Replacing ${TO_REPLACE} with ${HASH} in ${f}..."
+      sed -i "s/${TO_REPLACE}/${HASH}/g" ${f}
+
+      # will do nothing if there's nothing to add
+      git add ${f}
+
+      # will fail if no change: ignore it
+      git commit --message "Auto update hash in ${f} to ${HASH}" || true
```
Comment (rtitle) [inline scripts/version_update.sh:38]: As long as these fail if there’s no changes there’s no possibility of an infinite loop right? (Travis continually committing and building)

Comment (jmthibault79) [reply scripts/version_update.sh:38]: Right.  (Testing this was entertaining)

Comment (helgridly) [reply scripts/version_update.sh:38]: this is glorious. i look forward to when someone overzealously optimizes this out and we add travis to the list of cloud services we've broken

**Review by helgridly — APPROVED** [review]

**Review by ansingh7115 — APPROVED** [review]
Seems legit

**Pull request merged**